pd/src/verify.rs

```diff
@@ -269,6 +290,12 @@ impl StatefulTransactionExt for PendingTransaction {
             new_notes: self.new_notes.clone(),
             spent_nullifiers: self.spent_nullifiers.clone(),
             delegation_changes,
+            undelegation_validators: self
+                .undelegations
+                .iter()
+                .cloned()
+                .map(|u| u.validator_identity)
```

Comment:
nit: I think this would be epsilon-more-efficient if these lines were swapped, so that we're just cloning the identities and not the whole undelegations.

Reply:
agreed, fixed